pyedflib/edfwriter.py

```diff
@@ -171,39 +176,38 @@ def sex2int(sex):
         raise ValueError(f"Unknown sex: '{sex}'")
 
 
-def gender2int(gender):
+def gender2int(gender: Union[int, str, None]) -> Optional[int]:
```

Comment:
same as above

Reply:
I guess I didn't explain myself clearly. What I meant was that the `gender2int` function should have the same type hint as `sex2int`.

Reply:
Sure!